roles/foreman_installer/tasks/install.yml

```diff
@@ -8,7 +8,7 @@
 
 - name: 'Join options'
   set_fact:
-    foreman_installer_options_joined: "{{ foreman_installer_options | join(' ') }} {{ foreman_installer_options_internal_use_only | join(' ') }}"
+    foreman_installer_options_joined: "{{ foreman_installer_options | difference([] if foreman_installer_version is version('3.1', '>=') else ['--foreman-proxy-content-enable-ostree true']) | join(' ') }} {{ foreman_installer_options_internal_use_only | join(' ') }}"
```

Comment:
Is this also something that should be applied to `upgrade.yml`?

Reply:
no, because that is calling `install.yml` again, not joining the options itself.